package.json

```diff
@@ -61,9 +61,7 @@
     ]
   },
   "dependencies": {
-    "babel-core": "^6.24.1",
     "babel-plugin-istanbul": "^4.1.4",
-    "babel-plugin-transform-es2015-modules-commonjs": "^6.24.1",
```

Comment:
We use these two directly in ts-jest so I'm not sure about this. 

@GeeWee felt the need to include these as dependencies which makes me think this is needed

Reply:
@kulshekhar I think you are right. I just tried to test it by installing locally the package on this branch and I couldn't see `babel-plugin-transform-es2015-modules-commonjs`. Only `babel-core` was installed by Jest. That is a bit strange

Reply:
The rest of the changes look good to me so we can wait for @GeeWee to chime in on this

Reply:
I personally think that if we're importing and using something, that should explicitly be added as a dependency.

Reply:
hmm make sense 👍